frontend/src/utils/interfaces.ts

```diff
@@ -44,4 +44,31 @@ export interface ReusableInputProps {
   options?: string[];
   rows?: number;
   className?: string;
+}
+
+export interface ModalProps {
+  isVisible: boolean;
+  onClose: () => void;
+  children: React.ReactNode;
+}
+
+export interface DelModalProps {
```

Comment:
I am not sure what is `DelModalProps`?

Here you have `DelModalProps` and `DeleteModalProps`! What's the differences?